bigtable/google/cloud/bigtable/row.py

```diff
@@ -34,6 +37,13 @@
 """The maximum number of mutations that a row can accumulate."""
 
 
+def wrapped_partial(func, *args, **kwargs):
```

Comment:
Can you explain why this is needed?

Reply:
The Retry logic requires a callable method. By using a wrapped_partial we also get the encapsulation of the arguments for the retry method, call_mutate_row. This functionality is provided by functools.